pre-docs/applying_manifest_blocks.md

```diff
@@ -276,9 +276,30 @@ Create a manifest that sets up a web server with IIS and run it as a plan.
 
     The page displays the text **hello!**
 
+## Using Puppet Device modules from an apply statement
 
-**Related information**
+**Support for device modules is experimental and may experience breaking changes in y releases**
+
+Puppet device modules based on remote transports allow network devices and
+other targets that cannot run a puppet agent to be managed from a proxy. To use
+device modules from an apply statements the devices need to be added to the
+bolt inventory as remote targets. The `name` of the target will be used to
+auto-populate the `name`, `uri`, `user`, `password`, `host`, and `port` fields
+of the remote transport's connnection info. You must set the `remote-transport`
+option and any other connnection info under the `remote` section of config.
+
+```
+---
+nodes:
+  - name: "https://username:password@panos-device.example.com"
+    config:
+      transport: remote
+      remote:
+        address: panos-device-example.com
```

Comment:
For panos 1.0.0 they changed `address` to `host`. It can be a hostname or an IP. Specifying user and password in the URI is ok. I pretty much tried every combination of the config options in the snipped below from an inventoryfile. 
```yaml
  - name: http://admin:admin@xo9xrk9blv97nbs.delivery.puppetlabs.net
    alias: panos-8
    config:
      transport: remote
      remote:
        remote-transport: panos
        run-on: puppet-6
        host: xo9xrk9blv97nbs.delivery.puppetlabs.net
        # host: 10.32.115.95
        # user: admin
        # password: admin
        # apikey: LUFRPT14MW5xOEo1R09KVlBZNnpnemh0VHRBOWl6TGM9bXcwM3JHUGVhRlNiY0dCR0srNERUQT09
        ssl: false
```

Reply:
Note: Setting `host` does not make sense as it will always get overwritten by the host in the node name.